library/general/requiredlabels/requiredlabels.test.rego

```diff
@@ -0,0 +1,12 @@
+package k8srequiredlabels
```

Comment:
This duplicates the existing test file `src/general/requiredlabels/src_test.rego`.

Reply:
fixed